src/node/cli.ts

```diff
@@ -50,6 +50,7 @@ export interface Args extends VsArgs {
   "show-versions"?: boolean
   "uninstall-extension"?: string[]
   "proxy-domain"?: string[]
+  "proxy-path-passthrough"?: boolean
```

Comment:
Should we add a test to `cli.test.ts`?

Reply:
Will do!

Reply:
nvm, I think in this case it's super simple, there's really no room for error. the existing cli tests test more involved logic and complex flags.